packages/compiler/cli.js

```diff
@@ -46,6 +47,10 @@ const argv = yargs
         desc: "use KssPlugin for .(css|less|scss) files",
         type: "boolean",
     })
+    .option("out", {
```

Comment:
adding this option allows us to log things to the console and still use the CLI to output a JSON file (previously, we were just using stdout in the docs package of this repo)